docs/source/workflows/a2a/a2a-cli.md

```
@@ -0,0 +1,161 @@
+
+### Get Agent Info
+
+Get agent metadata including name, version, provider, and capabilities.
+
+**Usage:**
+```bash
+nat a2a client get_info --url $A2A_SERVER_URL
+```
+
+**Output example:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)